### PR TITLE
Fake cam2

### PR DESCRIPTION
**Pull request by moreApi**

### CODE REVIEW

File: DeviceAdapters/FakeCamera/FakeCamera.cpp [hunk 1]
```diff
@@ -835,13 +875,40 @@ void FakeCamera::getImg() const
...
 
+	roiHeight_ = std::max(1, std::min(cameraHeight_, img.rows));
+	roiWidth_ = std::max(1, std::min(cameraWidth_, img.cols));
 	updateROI();
 }
 
 void FakeCamera::updateROI() const
 {
-	roi_ = curImg_(cv::Range(roiY_, roiY_ + roiHeight_), cv::Range(roiX_, roiX_ + roiWidth_));
+	//roi_ = curImg_(cv::Range(roiY_, roiY_ + roiHeight_), cv::Range(roiX_, roiX_ + roiWidth_));
```
Comment (moreApi) [inline DeviceAdapters/FakeCamera/FakeCamera.cpp]: clean up pls

File: DeviceAdapters/FakeCamera/FakeCamera.cpp [hunk 2]
```diff
@@ -58,9 +62,13 @@ FakeCamera::FakeCamera() :
 	CreateProperty("Tiff Stack", "0", MM::Integer, false, new CPropertyAction(this, &FakeCamera::OnTiffStack));
 	SetAllowedValues("Tiff Stack", allowedValues);
 
+	//allowedValues.clear();
```
Comment (moreApi) [inline DeviceAdapters/FakeCamera/FakeCamera.cpp]: clean up pls
Do you have an idea what that did? I cant recreate my reasoning for clearing a vector that will not be used again :D

**Review by moreApi — COMMENTED** [review]
Looks good. Please remove the to unused comments and remove those JSON files and put their folder in the git ignore with a comment to this
VS bug https://developercommunity.visualstudio.com/t/JSON-Schema-being-written-to-solution-fo/10158717?ref=native&refTime=1670770713838&refUserId=0df77896-7743-4fe4-b1b6-e8ab3bf9f21b

**Comment by moreApi** [comment]
The JSON files are still part of the PR @PowerOfNames. You have to delete them and commit the deletation.

File: DeviceAdapters/FakeCamera/FakeCamera.cpp (hunk 2 above)
Comment (PowerOfNames) [reply DeviceAdapters/FakeCamera/FakeCamera.cpp]: I added this line cause I saw it somewhere else, but did not  need it anyway. its removed now!

**Pull request merged**